app/javascript/plain-javascript/page-reloader.js

```diff
@@ -3,14 +3,14 @@ document.addEventListener('DOMContentLoaded', () => {
   console.log('Loaded')
   const timerElement = document.querySelector('[data-reload-time]')
   if (timerElement) {
-    let timer = parseInt(timerElement.dataset['reloadTime'])
+    let timer = parseInt(timerElement.dataset['reloadTime'], 10)
```

Comment:
What was the problem here? Is it making it longer, because it was taking a while to do the submission?

Reply:
No change, it will behave the same. Is a "good practice" change for parseInt(), so it declares the string is a base 10 number.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt